Makefile

```diff
@@ -9,7 +9,7 @@ BASEDIR = $(shell pwd)
 TERMINAL_DIR = terminal
 DOJO_API_DIR = api
 DOJO_DMC_DIR = dmc
-ELWOOD_DIR = elwood
+# ELWOOD_DIR = elwood
```

Comment:
Elwood not part of stack anymore. Before this change doing rebuild-all (and other *all commands) finished with an error message stating that elwood dir did not exist.